Files/IShellPage.cs

```diff
@@ -46,6 +46,10 @@ public interface IShellPage : ITabItemContent, IMultiPaneInfo, IDisposable
 
     public interface IPaneHolder : IDisposable
     {
+        public event EventHandler ActivePaneChanged;
+        public IShellPage ActivePane { get; set; }
```

Comment:
Is this confirmed that it doesn't create circular references?

Reply:
@d2dyno1 No, it wouldn't create one because the lines you referenced merely show exposing ActivePane to be an interface member. Anyways, circular references are not likely to be the cause of our memory use problem.

Reply:
> No, it wouldn't create one because the lines you referenced merely show exposing ActivePane to be an interface member. 

`IPaneHolder` is a member of `IMultiPaneInfo` and `IShellPage` inherits from `IKultipaneInfo` so the `IShellPage` is accessible.

Reply:
@d2dyno1 It doesn't create a new one. I only made the existing one accessible to the interface. Nothing new to see here because (like I just stated) circular references are likely not the cause of our memory leak issue.